ui/app/styles/_layout.scss

```diff
@@ -0,0 +1,137 @@
+body {
+  @include Layout.stack(0);
+  color: rgb(var(--text));
+  background: rgb(var(--background));
+  min-height: 100vh;
+}
+
+#page {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width: 100vw;
+  max-width: 1600px;
+  margin: 0 auto;
+  padding: scale.$lg-3 scale.$lg-4 scale.$lg-5;
+}
+
+.svg-patterns {
+  display: none;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  padding-bottom: scale.$sm-3;
```

Comment:
these need to use margin-bottom, not padding: 

<img width="1062" alt="Screen Shot 2021-06-28 at 16 29 05" src="https://user-images.githubusercontent.com/1416421/123655345-2de55980-d82f-11eb-89f9-3b1439774019.png">